src/fu-tool.c

```diff
@@ -837,6 +838,9 @@ fu_util_install_blob(FuUtilPrivate *priv, gchar **values, GError **error)
 		return FALSE;
 	}
 
+	/* progress */
+	fu_progress_set_profile(progress, g_getenv("FWUPD_VERBOSE") != NULL);
+
```

Comment:
although I definitely agree we want this for verbose mode, what about adding a new `--profile` option that also does it?  My thought then it will be less verbose for a less technical contributor to say my values were way off and pull numbers out.

Reply:
Maybe? I'm not a far of more options generally, and in the *plugin development* case we'll be using `fwupdtool --plugins foobarbaz get-devices --verbose` anyway.

Reply:
OK - maybe this should be a phase 2 type of idea then if you want a feedback loop or anything like that.